ITE/4/README.adoc

```diff
@@ -351,10 +354,10 @@ main repository.
 }
 ```
 
-The materials aren't limited to the commits that make up a pull request, but can
-also contain other elements such as reviews or other comments that make up a
-discussion, as well as approvals from maintainers. It really depends on how the
-implementor, perhaps GitHub, chooses to define the components of a pull request.
+The materials are not limited to the commits that make up a pull request. They can
+also contain other elements such as reviews, or comments, as well as approvals from maintainers.
+It really depends on how the
+implementer, or perhaps GitHub, chooses to define the components of a pull request.
```

Comment:
GitHub would be the implementer here.

Reply:
Again, if you can make your comment a suggestion, I could just accept it.

Reply:
Resolved. (I'm unable to mark it for some reason)